### ISSUE TITLE
Errors thrown by call to missing setAccountMoneyAndRespect

### ISSUE THREAD

**Issue by dirtysouthgamingdev** [IT07/ArmA3_exile_vemf_reloaded#1]
Not sure, but guessing "setAccountMoneyAndRespect" was removed in Exile 0.9.8 and VEMFr is not updated to account for this.

Throws errors such as the following in extdb/logs, and presumably does not update account money and/or respect:

```
 [Thread 70264] extDB2: SQL_CUSTOM_V2: Error No Custom Call Not Found: Input String setAccountMoneyAndRespect:0:7183:76561198086821095
[08:19:12:717368 --7:00] [Thread 70264] extDB2: SQL_CUSTOM_V2: Error No Custom Call Not Found: Callname setAccountMoneyAndRespect
```

**Comment by IT07** [comment]
Thank you for reporting that error. I will investigate the issue and attempt to fix it.

**Comment by dirtysouthgamingdev** [comment]
This might be helpful - haven't been able to test.

I believe the single call to `setAccountMoneyAndRespect`needs to be changed to two separate calls for Exile 0.9.8. For example, in `aiKilled.sqf`, change:

`format["setAccountMoneyAndRespect:%1:%2:%3", _k getVariable ["ExileMoney", 0], _nwRspct, (getPlayerUID _k)] call ExileServer_system_database_query_fireAndForget;`

to 

```
format['setAccountScore:%1:%2', _nwRspct, (getPlayerUID _k)] call ExileServer_system_database_query_fireAndForget; 
format['setPlayerMoney:%1:%2', _k getVariable ["ExileMoney", 0], (getPlayerUID _k)] call ExileServer_system_database_query_fireAndForget; 
```

**Comment by IT07** [comment]
Ok thank you for that piece of code. I will test it out as soon as I can.

**Comment by IT07** [comment]
Issue has been resolved in version 0745.1
